library/src/interactor_impl.cxx

```diff
@@ -244,10 +245,19 @@ class interactor_impl::internals
   {
     internals* self = static_cast<internals*>(clientData);
     vtkStringArray* filesArr = static_cast<vtkStringArray*>(callData);
+    const std::regex charsToEscape(R"(([\"\\]))");
```

Comment:
```suggestion
    const std::regex charsToEscape(R"((["\\]))");
```
shouldn't you also drop the first backslash when going from `"([\"\\\\])"` to an `R"` string?

Reply:
absolutely !